scout/server/links.py

```diff
@@ -908,18 +906,6 @@ def hmtvar_link(variant_obj):
     return url_template.format(id=variant_obj.get("hmtvar_variant_id"))
 
 
-def spidex_human(variant_obj):
```

Comment:
This function needs to be called to fix a variant before  it's shown on the report (otherwise spidex is a very long number with little immediate meaning). The problem is that in the code it is not invoked by some variant decorator (easily accessible from the general report code) but from this function, which does many other things to the variant (not useful at all when creating a variant report):

<img width="805" alt="image" src="https://github.com/user-attachments/assets/f232f0c9-aa68-4a8c-82ce-d43e6b72fe2d" />

For this reason I've removed this function from here and simply added it to the jinja filters callable from any template.